spark/src/test/scala/org/apache/spark/sql/delta/uniform/UniFormE2ESuite.scala

```diff
@@ -83,7 +83,7 @@ abstract class UniFormE2EIcebergSuiteBase extends UniFormE2ETest {
             s"where ${partitionColumnName}=${partitionColumnsAndValues._2}"
           // Verify against Delta read and Iceberg read
           checkAnswer(spark.sql(verificationQuery), Seq(Row(123)))
-          checkAnswer(createReaderSparkSession.sql(verificationQuery), Seq(Row(123)))
```

Comment:
it's important to go through the `read` function, for some reason when I go through `createReaderSparkSession` directly that doesn't actually use the Iceberg spark session even though we are overriding it correctly.